core/src/main/java/hudson/cli/CliProtocol.java

```diff
@@ -25,7 +26,7 @@
  * @author Kohsuke Kawaguchi
  * @since 1.467
  */
-@Extension
+@Extension @Symbol("cli")
```

Comment:
See [@Symbol javadoc](https://github.com/jenkinsci/structs-plugin/blob/master/annotation/src/main/java/org/jenkinsci/Symbol.java#L21-24) that clarifies this. (cc:@kzantow)